keyboards/playkbtw/pk64rgb/rules.mk

```diff
@@ -0,0 +1,28 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

Reply:
The firmware will be too large after modifying it into full bootmagic functions. Do I need to make this change?

Reply:
Full bootmagic is already removed, and `BOOTMAGIC_ENABLE = yes` actually enables Bootmagic Lite; the `lite` value is deprecated and scheduled to be removed.

https://docs.qmk.fm/#/ChangeLog/20210828?id=bootmagic-full-removal

Reply:
> Full bootmagic is already removed, and `BOOTMAGIC_ENABLE = yes` actually enables Bootmagic Lite; the `lite` value is deprecated and scheduled to be removed.
> 
> https://docs.qmk.fm/#/ChangeLog/20210828?id=bootmagic-full-removal

I see. Thanks for clarifying. :)